requirements.txt

```diff
@@ -1,9 +1,9 @@
 beautifulsoup4>=4.6.3
 ffmpy>=0.2.2
-le-utils>=0.1.14
+le-utils>=0.1.19
 matplotlib==2.2.3
 numpy==1.15.4
 Pillow==5.4.1
 youtube-dl>=2018.11.7
```

Comment:
Could you please change this to `youtube-dl>=2019.6.8`

I tried running tests locally in my py2.7 env and it had the old version so youtube tests fail. CI is passing because it downloads latest.